sql/schema/schema.sql

```diff
@@ -1370,6 +1370,10 @@ CREATE UNIQUE INDEX "Worker_id_key" ON "Worker" ("id" ASC);
 -- CreateIndex
 CREATE UNIQUE INDEX "Worker_webhookId_key" ON "Worker" ("webhookId" ASC);
 
+-- CreateIndex
+
+CREATE INDEX "Worker_lastHeartbeatAt_idx" ON "Worker" ("lastHeartbeatAt", "tenantId");
```

Comment:
The name of this index should be "Worker_tenantId_lastHeartbeatAt_idx", and I think we should swap the index `CREATE INDEX "Worker_lastHeartbeatAt_idx" ON "Worker" ("tenantId", "lastHeartbeatAt");`